app/src/main/res/drawable/ic_striga_kyc_approved.xml

```diff
@@ -0,0 +1,101 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
you sure it's a good idea to keep this big PNG as SVG?

Reply:
@gslevinkov I've checked, it doesn't have any artifacts etc, so why not? some of such big images we store in vector, and we don't face any problems with them